notes.md

```diff
@@ -28,3 +28,215 @@ About:
 - Cover things like time spent in a kitchen, learning to code
 - Talk about art and knitting - the creative stuff
 - Throw up gamertags and games?
+
+<section class="cards-wrapper">
```

Comment:
This code was from an example I liked, I lifted what I needed to create my version of this effect, that colors the card background starting from left corner.